components/websubhub-consolidator/consolidator_service.bal

```diff
@@ -39,46 +39,74 @@ isolated function consolidateSystemState() returns error? {
     do {
         while true {
             kafka:BytesConsumerRecord[] records = check conn:websubEventConsumer->poll(config:kafka.consumer.pollingInterval);
+            log:printDebug("Polled Kafka records for state consolidation", recordCount = records.length(), pollingInterval = config:kafka.consumer.pollingInterval);
```

Comment:
If we have any computation, it's better to use a function pointer, as it is only executed when we enable the debug level. Otherwise, it has unnecessary tasks.

```ballerina
log:printDebug("Polled Kafka records for state consolidation", 
                             recordCount = isolated function() returns int { return records.length();}, 
                             pollingInterval = config:kafka.consumer.pollingInterval);
```